### DISCUSSION TITLE
Column view

### DISCUSSION THREAD

**Discussion by Au-angeloupali** [spacedriveapp/spacedrive#2700] [Feature Request]
I beg you to let the update with the column view come out.
I am desperate I keep changing programmes, yours seems stable functional and beautiful to me.
I am begging you to include the column view so that I and many other users can use it on a daily basis.

What is the realistic release time?

many thanks, Angelo.

**Comment by iLynxcat** [comment]
Hiya, sorry for the delay on this Angelo. I was previously working on this (#2591) but unfortunately our team recently had to shift focus to some other issues & projects for the time being. I'll be back working on this as soon as possible, we expect to ship this with our full 1.0 release (or shortly after) since we know lots and lots of users have been requesting it. Thanks for your patience and please keep an eye on our releases 1.0 and beyond for it

**Reply by Au-angeloupali** [reply]
but I did not understand if this is a project (Spacedrive) you are constantly working on and I should expect updates soon, or if it is parked for the time being?
I am now using ‘Files’ but it has many stability problems.

Thank you Angelo.

**Reply by Au-angeloupali** [reply]
@iLynxcat it will be integrater also the acrylic effect (Win 11) with the next relase?

**Reply by Au-angeloupali** [reply]
for this section

![image](https://github.com/user-attachments/assets/de9e5f2c-090a-495d-9eca-80ce5f5f0a13)

**Comment by Au-angeloupali** [comment]
@iLynxcat :(

**Comment by Au-angeloupali** [comment]
@iLynxcat I would like to know if the project is at a standstill so I can avoid daily grumbling if there is an update, thank you Angelo

**Reply by iLynxcat** [reply]
Hiya, apologies we’ve all been swamped with work on features for 0.5.0 & honestly haven’t paid enough attention to GitHub recently. Once we have the beta out we’ll be able to get back to better communicating here. Lots of moving pieces we’re working to put in place for that launch. We are in active development still, even though merges to main are currently slow we have lots of active branches we’re committing to every single day for the product, and some repos that aren’t public (at least yet). 

We’re not targeting to add the acrylic effect in 0.5.0 immediately but I am hoping to lead a larger design refresh later down the road toward 1.0. That redesign likely be where a Columns view will take place.

Thanks for your patience.

**Comment by Au-angeloupali** [comment]
Thank you for your work.
I put a lot of trust in spacedrive because I have not yet found a stable file explorer with the features I am looking for.
